app/src/main/kotlin/cloud/keyspace/android/StartHere.kt

```diff
@@ -375,15 +373,10 @@ class StartHere : AppCompatActivity() {
             loadingScreenFragmentView = inflater.inflate(R.layout.loading_screen, container, false)
 
             loadContent()
-            try {
-                generateCryptoObjects()
-                // logger()
-                if (mode == MODE_CREATE_ACCOUNT) createAccount()
-                else if (mode == MODE_SIGN_IN) signIn()
+            generateCryptoObjects()
 
-            } catch (unknownError: Exception) {
-                showCryptographyErrorDialog()
-            }
```

Comment:
Any reason we are removing exception handling here?

Reply:
I can't seem to replicate the "cryptographic error" dialog that some users get. If we get rid of this generic exception, we can crash the app manually and make users send us a bug report via email and figure it out that way. In any case, only two users reported this to us.

Reply:
Understood